invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/RunsLogs.js

```diff
@@ -0,0 +1,212 @@
+      }
+
+      const run = response.data;
+      run.formatted_started_at = this.formatDatetime(run.started_at);
+      const runDuration = this.getDurationInMinutes(run.started_at, run.finished_at);
+      this.setState({ run: run, runDuration: runDuration });
+      if (run.status === "SUCCESS" || run.status === "FAILED" || run.status === "PARTIAL_SUCCESS") {
+        clearInterval(this.logsInterval); // Stop fetching logs if run finihsed
+      }
+    } catch (err) {
+      console.error("Error checking run status:", err);
+      this.setState({ error: err.message });
+    }
+  };
+
+  componentDidMount() {
+    this.logsInterval = setInterval(async () => {
+      const { run, sort } = this.state;
+      if (run.status === "RUNNING") {
+        await this.fetchLogs(run.id, sort); // Fetch logs only if the run is running
+        await this.checkRunStatus(run.id, run.job_id); // Check the run status
+      }
+    }, 2000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.logsInterval);
+  }
+
+  render() {
+    const { error, logs, run, runDuration, sort } = this.state;
+    const levelClassMapping = {
+      DEBUG: "",
+      INFO: "primary",
+      WARNING: "warning",
+      ERROR: "negative",
+      CRITICAL: "negative",
+    };
+
+    const getClassForLogLevel = (level) => levelClassMapping[level] || "";
+    return (
+      <Container>
+      <Header as="h2" className="mt-20">
+        {run.title}
+      </Header>
+      <Divider />
+      {error && (
+        <Message negative>
+        <Message.Header>Error Fetching Logs</Message.Header>
+        <p>{error}</p>
+        </Message>
+      )}
+      <Grid celled>
+        <Grid.Row>
+        <Grid.Column width={3}>
+          <Header as="h4" color="grey">
+          Job run
+          </Header>
+          <List>
+          <List.Item>
+            <Icon
+            name={
+              run.status === "SUCCESS"
+              ? "check circle"
+              : run.status === "FAILED"
+                ? "times circle"
+                : run.status === "RUNNING"
+                ? "spinner"
+                : run.status === "PARTIAL_SUCCESS"
+                  ? "exclamation circle"
+                  : "clock outline"
+            }
+            color={
+              run.status === "SUCCESS"
+              ? "green"
+              : run.status === "FAILED"
+                ? "red"
+                : run.status === "RUNNING"
+                ? "blue"
+                : run.status === "PARTIAL_SUCCESS"
+                  ? "orange"
+                  : "grey"
+            }
+            />
+            <List.Content>
+            {run.formatted_started_at ? (
+              <>
+              <p>
+                <strong>{run.formatted_started_at}</strong>
+              </p>
+              <p className="description">{runDuration} mins</p>
+              </>
+            ) : (
+              <p className="description">Not yet started</p>
+            )}
```

Comment:
nit: are the paragraphs needed? 
with List.Item you should use List.Description, instead of "faking it" with the description name class
it will help with readability

Reply:
The reason to use the paragraphs is to do the line jumps and spacing, if we use List.Description we will need to add custom classes to it to manage this, let me know if I should update it 
![image](https://github.com/user-attachments/assets/1f809fc3-3483-4910-83b8-4cd9ddc11c86)